Subsytems/Elevator/SparkElevator.java

```diff
@@ -0,0 +1,69 @@
+package POPLib.Subsytems.Elevator;
+
+import com.revrobotics.spark.ClosedLoopSlot;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.SparkBase.ControlType;
+import POPLib.Control.FFConfig;
+import POPLib.Math.MathUtil;
+import POPLib.Motor.FollowerConfig;
+import POPLib.Motor.MotorConfig;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class SparkElevator extends Elevator {
+    SparkMax leadMotor;
+    SparkMax followMotor = null;
+
+    public SparkElevator(MotorConfig motorConfig, FFConfig ffConfig, boolean tuningMode, String subsystemName) {
+        super(ffConfig, tuningMode, subsystemName);
+
+        leadMotor = motorConfig.createSparkMax();
+        leadMotor.getEncoder().setPosition(0.0);
+    }
+
+    public SparkElevator(MotorConfig motorConfig, FollowerConfig followerConfig, FFConfig ffConfig, boolean tuningMode, String subsystemName) {
+        this(motorConfig, ffConfig, tuningMode, subsystemName);
+
+        followMotor = followerConfig.createSparkMax();
+        followMotor.getEncoder().setPosition(0.0);
+    }
+
+    public void periodic() {
+        super.tuning.updatePID(leadMotor);
+        leadMotor.getClosedLoopController().setReference(
+            super.setpoint.get(), 
+            ControlType.kPosition, 
+            ClosedLoopSlot.kSlot1, 
+            super.feedforward.calculate(getEncoderPos(), 0.0));
+        SmartDashboard.putNumber("Elevator lead motor pos", getEncoderPos());
+        if (followMotor != null) {
+            SmartDashboard.putNumber("Elevator follow motor pos", followMotor.getEncoder().getPosition());
+        }
+    }
+
+    public double getEncoderPos() {
+        return leadMotor.getEncoder().getPosition();
+    }
+
+    public double getError(double setpoint) {
+        return MathUtil.getError(leadMotor, setpoint);
+    }
+
+    public Command moveUp(double speed) {
```

Comment:
These two command won't work if the periodic runs, consider removing them or making them function in some way that the periodic doesn't inhibit